### DISCUSSION TITLE
如何修改select组件的 高度?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#41425] [Q&A] [answered]
希望调整高度为30, 现在size: default 为32, 我应该通过样式修改还是配置size,还是全局less变量什么的, 有方法吗? 谢谢

**Comment by afc163** [comment, marked as answer]
<img width="957" alt="图片" src="https://user-images.githubusercontent.com/507615/227180340-c2a1e007-e83a-48a0-b625-37708100e912.png">

**Comment by afc163** [comment]
<img width="703" alt="图片" src="https://user-images.githubusercontent.com/507615/227181124-caf39431-80ef-4500-9670-6b419c02836c.png">

**Reply by vaynevayne** [reply]
你这是gpt-3 吧, 都不对 
这个paddingY 还占着空间, 必须去掉, 有点头疼
<img width="1306" alt="image" src="https://user-images.githubusercontent.com/124666577/227183482-e56495d2-5049-4ba7-865f-ab569bfbc6b2.png">

**Comment by afc163** [comment]
GPT 的答案不太对。https://codesandbox.io/s/ji-ben-shi-yong-antd-5-3-2-forked-4lu41i?file=/demo.tsx

```jsx
  <ConfigProvider
    theme={{
      components: {
        Select: {
          controlHeight: 30
        }
      }
    }}
  >
    <Select
      defaultValue="lucy"
      style={{ width: 120 }}
      onChange={handleChange}
      options={[
        { value: "jack", label: "Jack" },
        { value: "lucy", label: "Lucy" },
        { value: "Yiminghe", label: "yiminghe" },
        { value: "disabled", label: "Disabled", disabled: true }
      ]}
    />
  </ConfigProvider>
```

**Reply by dxall** [reply]
這是5的api，4該怎麼改呢

**Reply by vaynevayne** [reply]
还是支持 classNames 好啊
